src/Widgets/SpaceWidget/SpaceWidgetEditingConfig.js

```diff
@@ -0,0 +1,16 @@
+import * as Scrivito from "scrivito";
+import spaceWidgetIcon from "../../assets/images/space_widget.svg";
+
+Scrivito.provideEditingConfig("SpaceWidget", {
+  title: "Space",
+  thumbnail: spaceWidgetIcon,
+  attributes: {
+    size: {
+      title: "Choose size",
+    },
+  },
+  properties: ["size"],
+  initialContent: {
+    size: 1,
```

Comment:
I think `5` is a better default, then `1`:

```suggestion
    size: 5,
```